scripts/release.sh

```diff
@@ -206,12 +207,38 @@ if [ -n "$OVERRIDE_COMMIT_HASH" ]; then
   alias_tag=
 fi
 
+# This variable will be used to flag if the helm chart dependencies have been
+# been updated.
+_helm_dependencies_updated=false
 for name in $IMAGES; do
   image_basename=$($NIX_EVAL -f . images.$BUILD_TYPE.$name.imageName | xargs)
   image=$image_basename
   if [ -n "$REGISTRY" ]; then
     image="${REGISTRY}/${image}"
   fi
+
+  if [ "$_helm_dependencies_updated" = false ]; then
+    for helm_chart_user in ${IMAGES_THAT_REQUIRE_HELM_CHART[@]}; do
+      if [ "$name" = "$helm_chart_user" ]; then
+        echo "Updating helm chart dependencies..."
+        # Get script directory
+        SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
On the control-plane we did this:
SCRIPTDIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"

Could add it to the top of the file?

Reply:
Using
```bash
SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
```